frontend/src/metabase/components/UserHasSeen/UserHasSeenAll.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Patterns/UserHasSeen/UserHasSeenAll" />
```

Comment:
I wonder if we could just call this "Product Call Outs" just to make it clearer what this is when I'm flipping around storybook.